src/main/ruby/codacy/rubocop/generator.rb

```diff
@@ -105,8 +119,9 @@ def self.run(file_path = "rubocop-doc.yml")
             patternId:  cop_data[:name].gsub("/", "_"),
             level:      level(cop_data),
             category:   category(cop_data),
+            subcategory: subcategory(cop_data),
             parameters: parameters(cop_data)
-          }
+          }.delete_if { |key, value| value == nil }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer the use of the `nil?` predicate.](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=5072804)